pkg/storage/types/pvc.go

```diff
@@ -46,6 +52,10 @@ func (e PvcNotFoundError) Error() string {
 	return e.Reason
 }
 
+func IsDataVolumeGarbageCollected(pvc *k8sv1.PersistentVolumeClaim) bool {
```

Comment:
nit: If you don't want to check if the pvc is nil, then I would not declare an additional function, but check directly the condition there